GliaWidgets/Public/Glia/Glia.swift

```diff
@@ -126,10 +126,11 @@ public class Glia {
         interactor = createdInteractor
 
         if let callback = completion {
-            createdInteractor.withConfiguration { [weak createdInteractor] in
+            createdInteractor.withConfiguration { [weak createdInteractor, weak self] in
                 guard let interactor = createdInteractor else { return }
 
                 // TODO: Configure string providing from Core SDK here.
+                self?.stringProviding = .init(getRemoteString: GliaCore.sharedInstance.localeProvider.getRemoteString)
```

Comment:
[Same](https://github.com/salemove/ios-sdk-widgets/pull/779/files#r1333081122).

Reply:
Now it's fixed 😄 2550ea25f4220bad6f2283d72d39110572d42692